File: Pratica 6 - Funcoes 2/cpp_logic/exercicio3_v2.cpp
#include <iostream>
using namespace std;

bool validar(string senha) {
    return senha.length() >= 8;
}

int main() {
    string senha = "";

    while (!validar(senha)) {
        cout << "Digite uma senha: ";
        cin >> senha;
    }

    cout << "Senha válida!";
    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio3_v2.java
import java.util.Scanner;

public class Exercicio3_v2 {
    static boolean validar_senha(String senha) {
        return senha.length() >= 8;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String senha = "";

        while (!validar_senha(senha)) {
            System.out.println("Digite uma senha: ");
            senha = sc.nextLine();
        }

        System.out.println("Senha válida!");
        sc.close();
    }
}
